Sources/TIM/AppAuth/AppAuthControllerMock.swift

```diff
@@ -13,7 +13,7 @@ final class AppAuthControllerMock: OpenIDConnectController {
     private var rt: JWTString?
     private var at: JWTString?
 
-    func login(presentingViewController: UIViewController, completion: @escaping ((Result<JWT, TIMAuthError>) -> Void), didCancel: (() -> Void)?, willPresentSafariViewController: ((SFSafariViewController) -> Void)?, shouldAnimate: (() -> Bool)?, authorizationRequestNonce: String?) {
+    func login(presentingViewController: UIViewController?, completion: @escaping ((Result<JWT, TIMAuthError>) -> Void), didCancel: (() -> Void)?, willPresentSafariViewController: ((SFSafariViewController) -> Void)?, shouldAnimate: (() -> Bool)?, authorizationRequestNonce: String?) {
```

Comment:
Making `presentingViewController` optional, as it's now not a given, that there's a viewController passed into the function, as if the user added a custom user agent in the configuration of TIM, it won't be needed. Right now the passed viewController will be the one presenting the login view - always with a push to a SFSafariViewController. This is not always ideal/wanted, hence this change.